SU-SWS/stanford_profile
# conditional fields

# READY FOR REVIEW

# Summary
- Use conditional fields instead of the hard coded states in the profile helper module.
- https://github.com/SU-SWS/stanford_profile_helper/pull/182

# Need Review By (Date)
- 3/15

# Urgency
- medium

# Steps to Test
1. `composer require su-sws/stanford_profile:dev-conditional-fields --no-update -n; composer update -n`
2. `blt drupal:update`
3. view the site lockup settings (`/admin/config/system/lockup-settings`) and local footer settings (`/admin/config/system/local-footer`)
4. verify the show/hide of the fields works as expected for the lockup settings.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
